service.tsx

```diff
@@ -0,0 +1,12 @@
+const service = {
+    getId: function ({ item }) {
```

Comment:
Do funkcji narzędziowych raczej przekazuj bezpośrednio to co chcesz obsluzyc. W typ przypadku jest to adres:
```suggestion
    getId: (url: string): number | undefined {
```